Packs/CarbonBlackDefense/ReleaseNotes/4_0_0.md

```diff
@@ -0,0 +1,31 @@
+****
+#### Integrations
+
+##### New: Carbon Black Endpoint Standard v3
+
+- Deprecated the **cbd-find-events** and **cbd-find-events-results** commands, replaced by **cbd-find-observation**.
+- Deprecated the **cbd-find-events-details** and **cbd-find-events-details-results** commands, replaced by **cbd-find-observation-details**.
+- Deprecated the **cbd-get-policies** command, replaced by **cbd-get-policies-summary**.
+
+- Updated the **cbd-find-processes** command to be a polling command which includes **cbd-find-processes-results**.
+
+- Added the **cbd-find-observation-results** command to manually retrieve observation results without polling.
+- Added the **cbd-find-observation-details-results** command to manually retrieve observation details results without polling.
+
+- Updated the **cbd-alerts-search** command context output; removed the **category** arg and updated **tag** arg to **tags**.
+
+- Updated the **cbd-get-policy**, **cbd-create-policy**, **cbd-update-policy**, and **cbd-set-policy** commands context output arguments to match the new API format
```

Comment:
Updated the **cbd-get-policy**, **cbd-create-policy**, **cbd-update-policy**, and **cbd-set-policy** commands context output arguments to match the new API format.